app/components/non_bordered_cards_component/non_bordered_cards_component.scss

```diff
@@ -42,12 +42,14 @@
 .non-bordered-card__image {
   width: 100%;
   padding-bottom: 0;
+
+  &--bottom-border {
+    border-bottom: 8px solid #F47B31;
+  } 
```

Comment:
```suggestion
```
Pls remove the orange `--bottom-border` CSS class as I don't think you used it.

Reply:
resolved thank you